### PR TITLE
openlineage: adjust log levels

### PR DESCRIPTION
**Pull request by JDarDagran**
OpenLineage provider is reported to be a bit too verbose. This PR attempts to lower logging levels of some messages and also cleans up the code.

### CODE REVIEW

File: airflow/providers/openlineage/extractors/base.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def extract(self) -> OperatorLineage | None:
             self.operator.__class__.__module__ + "." + self.operator.__class__.__name__
         )
         if fully_qualified_class_name in self.disabled_operators:
-            self.log.warning(
+            self.log.info(
```
Comment (kaxil) [inline airflow/providers/openlineage/extractors/base.py]: This should be DEBUG too, no? i.e user performed an action to disable the lineage for specific operators

File: airflow/providers/openlineage/extractors/base.py [hunk 2]
```diff
@@ -144,10 +144,10 @@ def _get_openlineage_facets(self, get_facets_method, *args) -> OperatorLineage |
                 job_facets=facets.job_facets,
             )
         except ImportError:
-            self.log.exception(
+            self.log.error(
```
Comment (kaxil) [inline airflow/providers/openlineage/extractors/base.py]:
```suggestion
            self.log.exception(
```
exception seems fine here

File: airflow/providers/openlineage/plugins/listener.py [hunk 3]
```diff
@@ -194,14 +194,14 @@ def on_dag_run_running(self, dag_run: DagRun, msg: str):
...
-            self.log.error("Executor have not started before `on_dag_run_success`")
+            self.log.warning("Executor have not started before `on_dag_run_success`")
             return
         self.executor.submit(self.adapter.dag_success, dag_run=dag_run, msg=msg)
 
     @hookimpl
     def on_dag_run_failed(self, dag_run: DagRun, msg: str):
         if not self.executor:
-            self.log.error("Executor have not started before `on_dag_run_failed`")
+            self.log.warning("Executor have not started before `on_dag_run_failed`")
```
Comment (kaxil) [inline airflow/providers/openlineage/plugins/listener.py]:
```suggestion
            self.log.debug("Executor have not started before `on_dag_run_failed`")
```

File: airflow/providers/openlineage/plugins/listener.py [hunk 4]
```diff
@@ -194,14 +194,14 @@ def on_dag_run_running(self, dag_run: DagRun, msg: str):
     @hookimpl
     def on_dag_run_success(self, dag_run: DagRun, msg: str):
         if not self.executor:
-            self.log.error("Executor have not started before `on_dag_run_success`")
+            self.log.warning("Executor have not started before `on_dag_run_success`")
```
Comment (kaxil) [inline airflow/providers/openlineage/plugins/listener.py]:
```suggestion
            self.log.debug("Executor have not started before `on_dag_run_success`")
```

File: airflow/providers/openlineage/extractors/base.py (hunk 1 above)
Comment (JDarDagran) [reply airflow/providers/openlineage/extractors/base.py]: Good catch.

**Comment by JDarDagran** [comment]
Thanks @kaxil ! I've commited changes as suggested.

**Review by kaxil — APPROVED** [review]

**Pull request merged**